### PR TITLE
Measures: DegreeIn, DegreeOut, DegreeInAv, DegreeOutAv

### PR DESCRIPTION
**Pull request by egolol**

### CODE REVIEW

File: braph2genesis/measures/_DegreeIn.gen.m [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+Measure.NONPARAMETRIC
+
+%%% ¡prop!
+COMPATIBLE_GRAPHS (constant, classlist) is the list of compatible graphs.
+%%%% ¡default!
+{'GraphWD' 'GraphBD' 'MultiplexWD' 'MultiplexBD'}
+
+
+%%% ¡prop!
+M (result, cell) is the degree-in.
```
Comment (hanzh346) [inline braph2genesis/measures/_DegreeIn.gen.m]: could it be M is the degreein, or M is a cell containing the degree-in value?

File: braph2genesis/measures/_DegreeInAv.gen.m [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+%%%% ¡default!
+Measure.NONPARAMETRIC
+
+%%% ¡prop!
+COMPATIBLE_GRAPHS (constant, classlist) is the list of compatible graphs.
+%%%% ¡default!
+{'GraphWD' 'GraphBD' 'MultiplexWD' 'MultiplexBD'}
+
+%%% ¡prop!
+M (result, cell) is the degreeinav.
```
Comment (hanzh346) [inline braph2genesis/measures/_DegreeInAv.gen.m]: could it be M is the degreeinav, or M is a cell containing the degree-inav value?

File: braph2genesis/measures/_DegreeOut.gen.m [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+Measure.NONPARAMETRIC
+
+%%% ¡prop!
+COMPATIBLE_GRAPHS (constant, classlist) is the list of compatible graphs.
+%%%% ¡default!
+{'GraphWD' 'GraphBD' 'MultiplexWD' 'MultiplexBD'}
+
+
+%%% ¡prop!
+M (result, cell) is the degreeout.
```
Comment (hanzh346) [inline braph2genesis/measures/_DegreeOut.gen.m]: could it be M is the degreeout, or M is a cell containing the degree-out value?

File: braph2genesis/measures/_DegreeOutAv.gen.m [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+%%%% ¡default!
+Measure.NONPARAMETRIC
+
+%%% ¡prop!
+COMPATIBLE_GRAPHS (constant, classlist) is the list of compatible graphs.
+%%%% ¡default!
+{'GraphWD' 'GraphBD' 'MultiplexWD' 'MultiplexBD'}
+
+%%% ¡prop!
+M (result, cell) is the degreeoutav.
```
Comment (hanzh346) [inline braph2genesis/measures/_DegreeOutAv.gen.m]: could it be M is the degreeoutav, or M is a cell containing the degree-out value?

**Review by bzufiriaki — APPROVED** [review]

**Review by JiaweiSun0512 — APPROVED** [review]

**Review by hanzh346 — APPROVED** [review]

**Pull request merged**